src/legacy/core_plugins/kibana/server/tutorials/register.js

```diff
@@ -81,66 +80,66 @@ import { consulMetricsSpecProvider } from './consul_metrics';
 import { cockroachdbMetricsSpecProvider } from './cockroachdb_metrics';
 
 export function registerTutorials(server) {
-  server.registerTutorial(systemLogsSpecProvider);
-  server.registerTutorial(systemMetricsSpecProvider);
-  server.registerTutorial(apacheLogsSpecProvider);
-  server.registerTutorial(apacheMetricsSpecProvider);
-  server.registerTutorial(elasticsearchLogsSpecProvider);
-  server.registerTutorial(iisLogsSpecProvider);
-  server.registerTutorial(kafkaLogsSpecProvider);
-  server.registerTutorial(logstashLogsSpecProvider);
-  server.registerTutorial(nginxLogsSpecProvider);
-  server.registerTutorial(nginxMetricsSpecProvider);
-  server.registerTutorial(mysqlLogsSpecProvider);
-  server.registerTutorial(mysqlMetricsSpecProvider);
-  server.registerTutorial(mongodbMetricsSpecProvider);
-  server.registerTutorial(osqueryLogsSpecProvider);
-  server.registerTutorial(phpfpmMetricsSpecProvider);
-  server.registerTutorial(postgresqlMetricsSpecProvider);
-  server.registerTutorial(postgresqlLogsSpecProvider);
-  server.registerTutorial(rabbitmqMetricsSpecProvider);
-  server.registerTutorial(redisLogsSpecProvider);
-  server.registerTutorial(redisMetricsSpecProvider);
-  server.registerTutorial(suricataLogsSpecProvider);
-  server.registerTutorial(dockerMetricsSpecProvider);
-  server.registerTutorial(kubernetesMetricsSpecProvider);
-  server.registerTutorial(uwsgiMetricsSpecProvider);
-  server.registerTutorial(netflowSpecProvider);
-  server.registerTutorial(traefikLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(systemLogsSpecProvider);
```

Comment:
These tutorial registrations need to remain separate at this stage because they are going to move to their specific plugins (e.g. the apm tutorial registration will move to the apm plugin (or one of their plugins). When we've identified all the non-specific tutorials, we can consider looping over them. If you happen to know which ones the new home plugin should house (the non-app specific ones), please let us know!